content-creators-kickstarter/exercise-types.md

```diff
@@ -0,0 +1,7 @@
+# Exercise types
+
+ 1. **Tutorial** \(created using the TFW\): the user is guided through the exercise, supported with hints and other ways based on the comprehensive messaging system of the TFW. This is the most frequently type of exercise used on the Avatao platform now. E.g.: [Tutorial Framework](https://next.avatao.com/events/getting-started-with-avatao/challenges/tutorial-framework/description)
+
+ 2. **Attack**: the user has to capture the flag. The flag is submited on the Avatao platform. This type of exercise has a simpler file structure, you can find examples in the template folder of the [Challenge Toolbox](https://github.com/avatao-content/challenge-toolbox/tree/v3/templates). E.g.: [Getting Started](https://next.avatao.com/events/getting-started-with-avatao/challenges/getting-started/description) 
+
+ 3. **Fix**: the user has to complete unfinished code or find and fix the vulnerability. This type of exercise also uses TFW for file editing, testing and reactive hinting based on the submitted code. E.g.: the last part of [A1 Injection](https://next.avatao.com/events/getting-started-with-avatao/challenges/a1-injection/description)
```

Comment:
In my opinion we shouldn't encourage users to create simply fixing exercises as they're not fun (again, in my opinion). Fixes should be part of tutorials in most cases. It's easier for the users and easier for us, because otherwise we have to memorize the dependencies between Attacking and Fixing exercises.

Reply:
Okay, since they are basically specialized tutorials, we can remove them from the docs.